plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/FeatureDevSessionContext.kt

```diff
@@ -127,11 +132,17 @@ class FeatureDevSessionContext(val project: Project, val maxProjectSizeBytes: Lo
             object : VirtualFileVisitor<Unit>() {
                 override fun visitFile(file: VirtualFile): Boolean {
                     val isFileIgnoredByExtension = runBlocking { ignoreFileByExtension(file) }
+                    val isDevFile = if (isAutoBuildFeatureEnabled == true) false else file.path.contains("devfile.yaml")
```

Comment:
per spec, `devfile.yml` is valid. please also unify with logic in 

https://github.com/aws/aws-toolkit-jetbrains/blob/7c89b1039aa9747392080a876f5c8780e22f3841/plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/caws/DevFileSchemaProviderFactory.kt#L20

and

https://github.com/aws/aws-toolkit-jetbrains/blob/7c89b1039aa9747392080a876f5c8780e22f3841/plugins/toolkit/jetbrains-ultimate/src/software/aws/toolkits/jetbrains/remoteDev/caws/DevfileWatcher.kt#L22

Reply:
i see, didn't know about this, let me change